package.json

```diff
@@ -86,8 +86,8 @@
     "nyc": "^11.4.1",
     "once": "^1.4.0",
     "pify": "^3.0.0",
-    "proxyquire": "^1.4.0",
     "request": "^2.83.0",
+    "retry-axios": "^0.3.2",
```

Comment:
Should this be moved to devDeps?

Reply:
I think it already is in `devDeps`.

Reply:
Ah, I was mistaken by the github UI. Sorry :)